temporal-sdk/src/main/java/io/temporal/internal/worker/LocalActivityExecutionContext.java

```diff
@@ -154,6 +161,14 @@ public boolean callback(LocalActivityResult result) {
     if (scheduleToCloseFuture != null) {
       scheduleToCloseFuture.cancel(false);
     }
+    SlotReleaseReason reason = SlotReleaseReason.taskComplete();
```

Comment:
Is it possible that the slot was never used?

Reply:
No - it's used in the sense that we tried to run the LA even if it timed out, which counts as `taskComplete` in this case. We might need finer-grained reasons at some point if people ask for them